### ISSUE TITLE
Standardize on HY_Features Associations

### ISSUE THREAD

**Issue by jvanulde** [NRCan/GSIP#35]
https://docs.opengeospatial.org/is/14-111r6/14-111r6.html

**Comment by jvanulde** [comment]
It appears that OGC is hosting defs: https://www.opengis.net/def/appschema/hy_features/hyf/HY_HydroFeature

@dblodgett-usgs @msondheim

**Comment by denevers** [comment]
Yes, but this does not address the persistant URI issue.   Or does it ?.  Don't remember if this went through OGC Naming authority.

**Comment by dblodgett-usgs** [comment]
That's also only the HY_HydroFeature package... we need the rest of the spec to get posted.

**Comment by jvanulde** [comment]
> That's also only the HY_HydroFeature package... we need the rest of the spec to get posted.

Should we contact the custodian at OGC and ask if they can expand it to include the rest of the spec? Seems to me that it's a good place to host it...

**Comment by denevers** [comment]
Dave is custodian of that spec I think

**Comment by dblodgett-usgs** [comment]
Rob Atkinson did the work to get what we have in the OGC-NA so far. I'm not sure where we left off. I'll contact Gobe and find out.

**Comment by dblodgett-usgs** [comment]
I followed up with Rob and he is going to look into finishing this up.
